src/course/en/course.json

```diff
@@ -54,7 +52,8 @@
                 "ariaRegion": "This component is a multiple choice question. Once you have selected an option select the submit button below."
             },
             "_media": {
-                "ariaRegion": "This is a media component which plays a video. Select the play / pause button to watch or listen. Alternatively you can select the link below for the transcript."
+                "ariaRegion": "This is a media component which plays a video. Select the play / pause button to watch or listen. Alternatively you can select the link below for the transcript.",
+                "transcriptButton": "Alternatively you can select the link below for the transcript."
```

Comment:
Is the duplication of "Alternatively you can select the link below for the transcript" correct? 

(And strictly speaking, it's a button not a link)